### PR TITLE
Web Page Design

### PR DESCRIPTION
**Pull request by LoboArkano**
In this pull request, we do a Youtube clone page.

### CODE REVIEW

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!

Here are the changes you need to make, before this project can be approved:
* [x] check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows one error in your project.
* [x] Pull Request should have meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
#### Navigation bar
* [x] Project should have a`<button>` tag that represents the search button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/search_button_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/)) Right now it is just an image.
* [x] Project should have at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/icons_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
#### Video section
* [x] Project should have like and dislike icons ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_like_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/)) Right now you have images there.
 * [x] `Share` should be a button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
* [x] `Subscribe` should be a button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_subscribe_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team       
#### Great Job so far, but there are a few more things required. 
### General
- [x] Pull request should have a meaning short description.
- [x] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload.

## Video section
 - [x] The share should be a button (screenshot from Pathwright)
-  [x] Subscribe should be a button with styled text not image (screenshot from Pathwright)

### Make changes and submit another code review request.
*Happy coding*

File: index.html [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+			</iframe>
+			
+			<article class="main_v_title">
+				<h6 id="video_name">Learn HTML5 and CSS3 From Scratch - Full Course</h6>
+				<div id="avatar">
+					<img src="images/avatar.png" alt="">
+				</div>
+				<div id="name_channel">freeCodeCamp.org</div>
+				<div id="subscribe_b">
+					<button style="border: none">
```
Comment (kobiyoyo) [inline index.html]: no inline CSS

File: index.html [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+                        <img src="images/sub_button.png" alt="">
+                    </button>
+				</div>
+				<div id="subscribers">1,67M</div>
+				<div id="views">150,384 views</div>
+				<div id="share">
+					<button style="border: none">
```
Comment (kobiyoyo) [inline index.html]: no inline CSS

**Review by mosesogwo — APPROVED** [review]
**Hi team. Good job making the requested changes:+1:. Your project is approved :heavy_check_mark:**.

You can now merge to master.

**Good luck in your next project:stars:**

**Pull request merged**